tasks/system_probe.py

```diff
@@ -609,14 +609,21 @@ def generate_cgo_types(ctx, windows=is_windows):
 
     for f in def_files:
         fdir, file = os.path.split(f)
+        absolute_input_file = os.path.abspath(f)
         base, _ = os.path.splitext(file)
         with ctx.cd(fdir):
+            output_file = "{base}_{platform}.go".format(base=base, platform=platform)
             ctx.run(
-                "go tool cgo -godefs -- -fsigned-char {file} > {base}_{platform}.go".format(
-                    file=file, base=base, platform=platform
-                )
+                "go tool cgo -godefs -- -fsigned-char {file} > {output_file}".format(file=file, output_file=output_file)
             )
-            ctx.run("gofmt -w -s {base}_{platform}.go".format(base=base, platform=platform))
+            ctx.run("gofmt -w -s {output_file}".format(output_file=output_file))
+            if replace_absolutes:
```

Comment:
Are you running from directories other than `/git/datadog-agent` and that is causing changes for you?

Reply:
If we do a one-time switch to the absolute paths, would that remove the need for this block?

Reply:
I don't think it would since multiple people could see different absolute paths depending on where the repo was cloned. But if you don't see the issue on your side maybe the problem is specific to my environment and in this case would not justify this change